src/Network/Network/NetworkWatcher/ConnectionMonitor/NewAzureNetworkWatcherConnectionMonitorEndPointObjectCommand.cs

```diff
@@ -31,6 +30,13 @@ public class NewAzureNetworkWatcherConnectionMonitorEndpointObjectCommand : Conn
         [ValidateNotNullOrEmpty]
         public string Name { get; set; }
 
+        [Parameter(
+            Mandatory = true,
+            HelpMessage = "The type of the connection monitor endpoint. Supported types are AzureVM, AzureVNet, AzureSubnet, ExternalAddress, MMAWorkspaceMachine, MMAWorkspaceNetwork.")]
+        [ValidateNotNullOrEmpty]
```

Comment:
@msJinLei , we discussed that it's not necessary, because the feature is in Public preview:( Not in GA. We need it to be in this release for Ignite.

Reply:
> @msJinLei , we discussed that it's not necessary, because the feature is in Public preview:( Not in GA. We need it to be in this release for Ignite.

@irrogozh 

The PR is targeting to master branch, in which breaking change is not allowed. 
- If you want to release these breaking change feature. we can only release it with a OOB release rather than next regular release. You need to create a preview branch (or with an existing branch) to include this PR and request for a OOB release after next regular release.
- If you want to merge these commits to master, you need wait for the major release.